app/client/css/style.css

```diff
@@ -9,26 +11,19 @@
 }
 
 
-
 #upload {
```

Comment:
When I run the app the upload button is hidden behind one of the video thumbnails, depending on the window size. I would increase the z-index of the upload button.